actgen/test.py

```diff
@@ -0,0 +1,121 @@
+import argparse
+import logging
+import random
+
+import numpy as np
+import gym
+import seeding
+import torch
+from tqdm import tqdm
+
+from . import utils
+from . import wrappers as wrap
+from .agents import RandomAgent, DQNAgent
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Trial:
+    def __init__(self, test=True):
+        args = self.parse_args()
+        self.params = self.load_hyperparams(args)
+        # hyper params to set up testing
+        self.params['max_env_steps'] = 10000
+        self.params['eval_every_n_steps'] = 100
+        self.params['epsilon_final'] = 0
+        self.params['n_eval_episodes'] = 10
+        self.params['replay_warmup_steps'] = 0
+        if self.params['test'] or test:
+            self.params['test'] = True
```

Comment:
remove

Reply:
is this meant for the line `self.params['test'] = True`? I thought we actually want to keep this for using `self.params['test']` on whether to load saved model or not

Reply:
Ah, you're right that one should stay.